bip-0003.md

```diff
@@ -72,10 +72,9 @@ Through its high visibility, it facilitates the community-wide consideration of
 source to retrieve the latest version of any BIP. The repository transparently records all changes to each BIP and
 allows any community member to retain a complete copy of the archive easily.
 
-The BIPs repository is not a tool to track acceptance[^acceptance], adoption, or community consensus on BIPs, beyond
-providing a brief overview of BIP statuses (see [Workflow](#workflow) below) to the audience.
-There is no formal or informal decision body that governs Bitcoin development or decides acceptance of BIPs. Bitcoin
-development emerges from the participation of stakeholders across the ecosystem.
+The BIPs repository neither tracks community sentiment nor ecosystem adoption[^adoption] of BIPs beyond
+the brief overview provided via the BIP status (see [Workflow](#workflow) below).
+No formal or informal decision body governs Bitcoin development or decides acceptance[^acceptance] of BIPs.
```

Comment:
Could "acceptance" be construed as acceptance into the BIPs repository?

```suggestion
No formal or informal decision body governs Bitcoin development or decides adoption[^adoption] of BIPs.
```

Reply:
Thanks, that’s a good suggestion. I amended this section to use "publication" (when a BIP is merged) and "adoption" (when a BIP is implemented by projects). "Acceptance" now only appears in a footnote commenting on community sentiment.